articles/ros2_threat_model.md

```diff
@@ -45,19 +52,78 @@ th {
 
 </div>
 
+## Table of Contents
+
+- [{{ page.title }}](#pagetitle)
```

Comment:
`{:toc}` ?

Reply:
The toc should appear automatically with that tag when running it with jekyll?

Reply:
I realize that I had this tag and it's not doing what I expect it to do. I guess it's fine to keep that manual for this PR but we should probably try to automate this somehow.

Reply:
We found out that putting a "* TOC" just above {:toc} generates a visible table of contents. See example below. You could just try! Visually it generates the _toc_ on the right side of the page (using Jekyll)

```suggestion
* TOC
{:toc}
```

Reply:
@olaldiko will you accept this? If so, maybe we should remove the TOC generated manually below?